index.html

```diff
@@ -149,142 +167,206 @@ <h2>Multi-Post Stories</h2>
-            <a href="#">JavaScript</a>
-          </li>
-          <li>
-            <img src="images/Ellipse1(1).jpg" alt="HTML" />
-            <a href="#">HTML</a>
-          </li>
-          <li>
-            <img src="images/Ellipse1(2).jpg" alt="CSS" />
-            <a href="#">CSS</a>
-          </li>
-        </ul>
-      </li>
-      <li class="knowledge">
-        <h2 class="kn_1">
-          <span>Frameworks</span>
-          <span>
-            <img src="images/arrow_right.png" alt="arrow_right">
-          </span>
-        </h2>
-      </li>
-      <li class="knowledge">
-        <h2 class="kn_1">
-          <span>Skills</span>
-          <span>
-            <img src="images/arrow_right.png" alt="arrow_right">
-          </span>
-        </h2>
-      </li>
-    </ul>
-  </section>
-
-  <section class="contact-form">
-    <div class="title-p">
-      <h1 class="contact-me">Contact me</h1>
-      <p class=p2>If you have an application you are interested in developing,
-         a feature that you need built or a project that needs coding.
-         I’d love to help with it </p>
-    </div>
-    <form  class="form" action="https://formspree.io/f/xbjwakbe" method="post">
-      <ul>
-        <li class="text">
-          <input required type="text" id="name" name="user_name" maxlength="30" placeholder="Write your name"/>
+    </li>
+    <li class="knowledge">
+      <h2 class="kn_1">
+        <span>Skills</span>
+        <span>
+          <img src="images/arrow_right.png" alt="arrow_right">
+        </span>
+      </h2>
+      <ul class="frameworks">
+        <li>
+          <img src="images/Ellipse_1.png" alt="JavaScript" />
+          <a href="">JavaScript</a>
         </li>
-        <li class="text">
-          <input required type="email" id="mail" name="user_email" placeholder="Write your email" />
+        <li>
+          <img src="images/Ellipse_2.png" alt="HTML" />
+          <a href="">HTML</a>
         </li>
-        <li class="text">
-          <textarea required id="msg" name="user_message" maxlength="500" placeholder="Write your message here"></textarea>
+        <li>
+         <img src="images/Ellipse_3.png" alt="CSS" />
+          <a href="">CSS</a>
         </li>
-      </ul> 
-      <input class="butt" type="submit" value="Get It Touch">
+      </ul>
+    </li>
+  </ul>
+</section>
+  <footer id="contact" class="contact">
+    <div class="contact-contact-container">
+    <div class="container-contact">
+      <h2 class="first-text-contact"> Contact me</h2>
+      <p class="second-text-contact">If you have an application you are interested in developing, a feature that you need built or a
+          project that
+          needs coding. I’d love to help with it </p>
+    </div>  
+    <form class="container-form" action="https://formspree.io/f/xbjwakbe" method="post">
+      <input id="name" class="text-area" type="text" name="name" maxlength="30" required placeholder="name">
```

Comment:
- [ ] Please set the background color to `#6070FF`
- [ ] Please delete the border-radius from the input fields
- [ ] Please center the `get in touch` button
- [ ] Please make sure that the placeholders inside the inputs have the same padding, and capitalized
- [ ] Contact-me container is   using  `margin: 100px 20%;` to layout, please use flexbox or grid to layout the content instead of margins and paddings. 
- [ ] Please decrease the space between the form inputs and the `<p>` tag

Your design  | Figma design
------------- | -------------
<img width="792" alt="Screenshot 2022-04-01 at 12 45 36" src="https://user-images.githubusercontent.com/77038610/161228710-8be31338-22db-4c70-a87e-cef3bdb26869.png">  | <img width="755" alt="Screenshot 2022-04-01 at 12 45 12" src="https://user-images.githubusercontent.com/77038610/161228643-e89e4eaa-e8ee-4479-ab69-e2de84758760.png">

- [ ] Finally, after you are done with the changes, please make sure that your design is similar to Figma